blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md

```diff
@@ -0,0 +1,772 @@
+---
+title: HummingBoard Pulse with Yocto and Buildroot
+abstract: 'In this article, I will show you how to build and run the image from
+           Yocto Project and SolidRun at HummingBoard Pulse. Furthermore, you
+           can find here examples of which tools and configurations were used.'
+cover: /covers/yocto-vs-buildroot.png
+author: cezary.sobczak
+layout: post
+published: true
+date: 2020-04-08
+archives: "2020"
+
+tags:
+  - SolidRun
+  - Buildroot
+  - HummingBoard
+  - Yocto
+  - u-boot
+categories:
+  - Miscellaneous
+  - OS Dev
+
+---
+
+## Introduction
+In this article, I will describe differences between two types of system
+images, i.e. SolidRun and Yocto. I was using a HummingBoard Pulse as a platform
+for my work. Wide description of a board you can found on
+[this](https://developer.solid-run.com/knowledge-base/hummingboard-pulse-getting-started/)
+website. After reading this, you will better understand which basics components
+are important for building own Linux distribution and how to use tools to do it.
+
+![hummingboard](/covers/hummboard.jpg)
+
+## Start work with a HummingBoard
+It's several things you need to possess:
+
+* Linux at your PC (because this way is easier)
+* 16GB Micro SD card
+* 12V Power adapter (the board has wide-range input of 7V-36V but 12V is
+recommended)
+* MicroUSB to USB for the console because the HummingBoard Pulse has an onboard
+FTDI chip, which means that there is no need to use external UART/USB converter
+* HummingBoard with SOM of course
+
+## Build and run a Buildroot image
+Buildroot was made by a SolidRun company as a target platform for their devices.
```

Comment:
It sounds like the Buildroot itself was made by the SolidRun. SolidRun have just added a target for their board to the Buildroot build system.